packages/block-editor/src/hooks/style.js

```diff
@@ -134,6 +133,126 @@ const skipSerializationPathsSave = {
  */
 const renamedFeatures = { gradients: 'gradient' };
 
+/**
+ * A utility function used to remove one or more paths from a style object.
+ * Works in a way similar to Lodash's `omit()`. See unit tests and examples below.
+ *
+ * It supports a single string path:
+ *
+ * ```
+ * omitStyle( { color: 'red' }, 'color' ); // {}
+ * ```
+ *
+ * or an array of paths:
+ *
+ * ```
+ * omitStyle( { color: 'red', background: '#fff' }, [ 'color', 'background' ] ); // {}
+ * ```
+ *
+ * It also allows you to specify paths at multiple levels in a string.
+ *
+ * ```
+ * omitStyle( { typography: { textDecoration: 'underline' } }, 'typography.textDecoration' ); // {}
+ * ```
+ *
+ * You can remove multiple paths at the same time:
+ *
+ * ```
+ * omitStyle(
+ * 		{
+ * 			typography: {
+ * 				textDecoration: 'underline',
+ * 				textTransform: 'uppercase',
+ * 			}
+ *		},
+ *      [
```

Comment:
```suggestion
 *		},
 *		[
```

Soothes whitespace OCD 💆‍♂️

Reply:
Fixed as part of https://github.com/WordPress/gutenberg/pull/45014/commits/ae0d16e03ac77ca165f9a4ec539a3258f15a0d88. I appreciate such OCDs 😉